frontend/glideinFrontendElement.py

```diff
@@ -1142,13 +1136,12 @@ def refresh_entry_token(self, glidein_el):
                     tkn_str = token_util.create_and_sign_token(
                         pwd_file, scope=scope, duration=duration, identity=identity
                     )
-                    # cmd = "/usr/sbin/frontend_condortoken %s" % glidein_site
-                    # tkn_str = subprocessSupport.iexe_cmd(cmd, useShell=True)
                     logSupport.log.debug("tkn_str= %s" % tkn_str)
```

Comment:
Let's comment this, adding something like: 
Sensitive information: uncomment only in development machines